sound/soc/sof/sof-audio.h

```diff
@@ -123,6 +127,7 @@ struct sof_ipc_pcm_ops {
 				   struct snd_pcm_substream *substream);
 	bool reset_hw_params_during_stop;
 	bool ipc_first_on_start;
+	bool delayed_platform_trigger;
```

Comment:
I'm not too happy about the `delayed_platform_trigger` name as it should describe that if it is set we are not going to stop/trigger the platform at trigger time, we are only going to do a STOP on hw_free().

Reply:
@ujfalusi @ranj063 yes the term 'delayed' is tainted in Linux. it hints at a delayed workqueue or something scheduled later, but that's not what is done at all.

stop_during_hwfree would be a more accurate description, no?